src/components/battles/animations/Ending.ts

```diff
@@ -85,6 +85,15 @@ export class Ending<TGameStartr extends FullScreenPokemon> extends GeneralCompon
             battleInfo.onComplete(outcome);
         }
 
+        if (battleInfo.endingtheme !== undefined && battleInfo.endingtheme !== "None") {
```

Comment:
Where does `"None"` come from? I would think just the `!== undefined` is sufficient. What if there's an actual audio theme named `"None"`?

Reply:
I was thinking None would be the case where they do not want any theme music playing after the battle. If they leave it undefined the battle theme would keep playing after the battle (which is also why I wanted the stopall there).

Reply:
Interesting. I think that warrants more discussion - can you file a followup issue about it & remove the code from the PR? We can always copy & paste it from the PR's commit history later.